airflow/providers/amazon/aws/hooks/athena.py

```diff
@@ -163,6 +213,7 @@ def poll_query_status(self, query_execution_id, max_tries=None):
                 break
             if max_tries and try_number >= max_tries:  # Break loop if max_tries reached
                 final_query_state = query_state
+                self.stop_query(query_execution_id)
```

Comment:
This line appears to be unrelated to fetching all results or not, right?

I guess this is probably a sane default, but this is a change in the current behaviour. Previously any query would continue running meaning that even if it takes a lot longer in Airflow it would (eventually) be available in the Athena console. 

Hmmm.

Reply:
Other than this (which I don't know if we want or not) the code looks good, and tests are always great :)

Reply:
@lindsable What do you think? Is this required? Is it a sane default? Are there cases where users might want the query to continue anyway?

Reply:
While I can't think of anything specific, I think you have a good point about changing existing functionality. We will leave it up to the end user to stop query if they choose. I'll remove this line.

Reply:
Can you ping me once this has been changed?

(Sorry this ticket has sat for _so_ long.)

Reply:
Changed! Just waiting for the tests to pass again. @ashb

Reply:
@ashb I'm not sure why the static checks are failing on the docker lint. Any insights?

Reply:
@lindsable Yeah does seem unrelated to your change -- try rebasing on to latest master.

Reply:
Looks like the same tests are also failing on master :(